source/Contribution-Guide/guide-examples/how-to-guide.html.md.erb

```diff
@@ -0,0 +1,33 @@
+---
+title: How-To Guide Template
+last_reviewed_on: 2025-02-25
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+## How to [Perform Specific Task] in [System/Application Name]
+This guide provides step-by-step instructions for [performing specific task] in [System/Application Name].
+
+## Prerequisites
+- Access level required for this task
+- Any necessary tools or software
+- Required background knowledge
+
+## How-to steps
+1. First step
+2. Second step
+3. Third step
+4. Fourth step
+
+## Verification
+How to verify that the task was completed successfully.
```

Comment:
Include who can verify if a tester is required e.g. a specific group/person to contact.

Reply:
added